### ISSUE TITLE
Error compiling nginx-clojure module with nginx

### ISSUE THREAD

**Issue by Foxly1990** [nginx-clojure/nginx-clojure#183]
Hi, `[nginx-1.13.3 and nginx-clojure-0.4.5]`
I get the following error when compiling nginx-clojure module with nginx:

```
nginx-clojure/src/c/ngx_http_clojure_mem.c: In function «nji_ngx_http_clojure_hijack_read_handler»:
nginx-clojure/src/c/ngx_http_clojure_mem.c:1921:19: error: this statement may fall through [-Werror=implicit-fallthrough=]
     wsctx->pstate = NGX_HTTP_CLOJURE_WEBSOCKET_PARSE_LEN;
nginx-clojure/src/c/ngx_http_clojure_mem.c:1923:4: note: here
    case NGX_HTTP_CLOJURE_WEBSOCKET_PARSE_LEN:
    ^~~~
nginx-clojure/src/c/ngx_http_clojure_mem.c:1938:19: error: this statement may fall through [-Werror=implicit-fallthrough=]
     wsctx->pstate = NGX_HTTP_CLOJURE_WEBSOCKET_PARSE_MASK;
nginx-clojure/src/c/ngx_http_clojure_mem.c:1940:4: note: here
    case NGX_HTTP_CLOJURE_WEBSOCKET_PARSE_MASK:
    ^~~~
nginx-clojure/src/c/ngx_http_clojure_mem.c:1946:19: error: this statement may fall through [-Werror=implicit-fallthrough=]
     wsctx->pstate = NGX_HTTP_CLOJURE_WEBSOCKET_PARSE_DATA;
nginx-clojure/src/c/ngx_http_clojure_mem.c:1948:4: note: here
    case NGX_HTTP_CLOJURE_WEBSOCKET_PARSE_DATA:

```

**Comment by xfeep** [comment]
Do you use gcc7?
Please try to add ` /*fallthrough*/` below the lines which got warning messages. e.g.
```c
case NGX_HTTP_CLOJURE_WEBSOCKET_PARSE_START:
.....
/*fallthrough*/
case NGX_HTTP_CLOJURE_WEBSOCKET_PARSE_LEN:
```

**Comment by brucekirkpatrick** [comment]
Thanks.   I just wanted to note that your comment regarding /*fallthrough*/ was the correct answer for me when I saw make fail in the same way on ubuntu 18.04.  There are only 6 occurrences of /*no break*/ that needed this other comment due to the newer compiler.

In these 2 files:
src/c/ngx_http_clojure_mem.c
src/c/ngx_http_clojure_shared_map_hashmap.c

**Comment by xfeep** [comment]
Fixed in v0.5.0.
